src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,133 @@
+
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.SerialPort.Port;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import com.kauailabs.navx.frc.AHRS;
+
+public class Drivetrain extends SubsystemBase {
+  private WPI_TalonFX driveLeftMotor1,driveLeftMotor2,driveRightMotor1,driveRightMotor2;
+  private AHRS gyro;
+
+  public Drivetrain() {
+
+    //creating motors
+    driveLeftMotor1 = new WPI_TalonFX(Constants.driveTalonLeftMotor1);
+    driveLeftMotor2 = new WPI_TalonFX(Constants.driveTalonLeftMotor2);
+    driveRightMotor1 = new WPI_TalonFX(Constants.driveTalonRightMotor1);
+    driveRightMotor2 = new WPI_TalonFX(Constants.driveTalonRightMotor2);
+
+    //making right motors go right
+    driveRightMotor1.setInverted(true);
+    driveRightMotor2.setInverted(true);
+    driveLeftMotor1.setInverted(false);
+    driveLeftMotor2.setInverted(false);
+
+    gyro = new AHRS(Port.kUSB);
+
+    resetGyro();
```

Comment:
You have to configure the encoders for them to work. Look at Talon.configSelectedSensor.